src/components/ItaliaTheme/GalleryPreview/GalleryPreview.jsx

```diff
@@ -74,7 +64,6 @@ const GalleryPreview = ({ id, viewIndex, setViewIndex, items }) => {
       {viewIndex != null && (
         <>
           <ModalHeader
-            closeButton={true}
```

Comment:
perchè abbiamo rimosso il bottone per chiudere?

Reply:
Il bottone si vede anche senza questa prop, anzi ho un errore in console se lo lasciamo qui

<img width="1876" alt="Screenshot 2023-11-08 alle 14 37 30" src="https://github.com/RedTurtle/design-comuni-plone-theme/assets/43245702/9800d80a-118a-45f9-8020-4ac194e368fb">

Ho visto che lo gestisce in automatico nel react-kit https://github.com/italia/design-react-kit/blob/master/src/Modal/ModalHeader.tsx